tools/resources/__init__.py

```diff
@@ -496,7 +496,10 @@ def _all_parents(self, file_path, base_path, into_path):
                 start_at = index + 1
                 break
         for n in range(start_at, len(components)):
-            parent_name = self._sep.join([into_path] + components[:n])
+            parent_name_parts = components[:n]
+            if into_path:
+                parent_name_parts.insert(0, into_path)
+            parent_name = self._sep.join(parent_name_parts)
```

Comment:
@theotherjimmy I believe this is the correct fix (see my commit message). If you have any concerns about this let me know.

Reply:
You're correct. That's the fix.